README.md

```diff
@@ -11,116 +11,30 @@ Capabilities:
 
 ## Install and Configure Prerequisites on the Cluster
 
-You'll need a DCOS cluster with one public node and at least five private nodes, DCOS CLI, and DCOS package CLIs.
+You'll need a DC/OS cluster with one public node and at least five private nodes, DC/OS CLI, and DC/OS package CLIs.
 
-Install package CLIs:
+Run `bash stage_ebc_demo.sh [master_ip] [ELB_Hostname]`
 
-```
-$ dcos package install cassandra --cli
-$ dcos package install kafka --cli
-```
+The above command will do the following:
 
-## Demo steps
-
-Install packages for DCOS UI:
-* kafka
-* cassandra
-* zeppelin
-* marathon-lb
-
-Wait until the Kafka and Cassandra services are healthly. You can check their status with:
-
-```
-$ dcos kafka connection
-...
-$ dcos cassandra connection
-...
-```
-
-## Edit the Tweeter Service Config
-
-Edit the `HAPROXY_0_VHOST` label in `tweeter.json` to match your public ELB hostname. Be sure to remove the leading `http://` and the trailing `/` For example:
-
-```json
-{
-  "labels": {
-    "HAPROXY_0_VHOST": "brenden-7-publicsl-1dnroe89snjkq-221614774.us-west-2.elb.amazonaws.com"
```

Comment:
I believe this step is still necessary if you want to have hostname resolution for the IP instead of IP. The CLI script doesnt use it is not strictly necessary and it is nontrivial to automatically detect the hostname